docs/index.rst

```diff
@@ -34,7 +34,8 @@ crowdfunding, blind auctions, multi-signature wallets and more.
 Translations
 ------------
 
-This documentation is translated into several languages by community volunteers, but the English version stands as a reference.
+This documentation is translated into several languages by community volunteers
+with varying degrees of completeness and actuality. The English version stands as a reference.
```

Comment:
I would say that especially in the context of this sentence, 'actuality' is overly complex, maybe 'accuracy' instead?